src/main/java/graphql/execution/preparsed/PreparsedDocumentEntry.java

```diff
@@ -24,20 +25,24 @@ public class PreparsedDocumentEntry implements Serializable {
     private final Document document;
     private final List<? extends GraphQLError> errors;
 
-    public PreparsedDocumentEntry(Document document) {
```

Comment:
right now  on masdter we have


```
    public PreparsedDocumentEntry(Document document) {
        assertNotNull(document);
        this.document = document;
        this.errors = null;
    }

    public PreparsedDocumentEntry(List<? extends GraphQLError> errors) {
        assertNotNull(errors);
        this.document = null;
        this.errors = errors;
    }

    public PreparsedDocumentEntry(GraphQLError error) {
        this(singletonList(assertNotNull(error)));
    }
```

This change gets rid of the `PreparsedDocumentEntry(List<? extends GraphQLError> errors)` contructor which is a SPI breaking change.  Please put it back so we can avoid an breaking change.  Sure the main code may not use it BUT any one using this today outside out code contexts will not be broken

Reply:
makes complete sense, let me add it back